numpy/polynomial/chebyshev.py

```diff
@@ -1647,6 +1647,130 @@ def chebfit(x, y, deg, rcond=None, full=False, w=None):
     return pu._fit(chebvander, x, y, deg, rcond, full, w)
 
 
+def chebfit2d(x, y, z, deg, rcond=None, full=False, w=None, max_degree=None):
+    """
+    2d Least squares fit of Chebyshev series to data.
+
+    Return the coefficients of a Chebyshev series of degree `deg` that is the
+    least squares fit to the data values `y` given at points `x`.
```

Comment:
```suggestion
    least squares fit to the data values `z` given at points `(x, y)`.
```

Reply:
This is why this repeated docstring is so dangerous, it almost ensures that one or many will be wrong.